proteuslib/tests/test_chlorination.py

```diff
@@ -0,0 +1,823 @@
+        model.fs.thermo_params.reaction_H2O_Kw.eps.value = eps
+        model.fs.thermo_params.reaction_NH4_Ka.eps.value = eps
+        model.fs.thermo_params.reaction_HOCl_Ka.eps.value = eps
+
+        # Equilibiurm reactions have eps in the 'rxn_params'
+        model.fs.rxn_params.reaction_NH2Cl_K.eps.value = eps
+        model.fs.rxn_params.reaction_NHCl2_K.eps.value = eps
+        model.fs.rxn_params.reaction_NCl3_K.eps.value = eps
+
+        assert model.fs.thermo_params.reaction_H2O_Kw.eps.value == eps
+        assert model.fs.thermo_params.reaction_NH4_Ka.eps.value == eps
+        assert model.fs.thermo_params.reaction_HOCl_Ka.eps.value == eps
+
+        assert model.fs.rxn_params.reaction_NH2Cl_K.eps.value == eps
+        assert model.fs.rxn_params.reaction_NHCl2_K.eps.value == eps
+        assert model.fs.rxn_params.reaction_NCl3_K.eps.value == eps
+
+
+
+    @pytest.mark.component
+    def test_scaling(self, chlorination_obj):
+        model = chlorination_obj
+        iscale.calculate_scaling_factors(model.fs.unit)
+
+        assert hasattr(model.fs.unit.control_volume, 'scaling_factor')
+        assert isinstance(model.fs.unit.control_volume.scaling_factor, Suffix)
+
+        assert hasattr(model.fs.unit.control_volume.properties_out[0.0], 'scaling_factor')
+        assert isinstance(model.fs.unit.control_volume.properties_out[0.0].scaling_factor, Suffix)
+
+        assert hasattr(model.fs.unit.control_volume.properties_in[0.0], 'scaling_factor')
+        assert isinstance(model.fs.unit.control_volume.properties_in[0.0].scaling_factor, Suffix)
+
+        assert hasattr(model.fs.unit.control_volume.reactions[0.0], 'scaling_factor')
+        assert isinstance(model.fs.unit.control_volume.reactions[0.0].scaling_factor, Suffix)
+
+    @pytest.mark.component
+    def test_initialize(self, chlorination_obj):
+        model = chlorination_obj
+
+        orig_fixed_vars = fixed_variables_set(model)
+        orig_act_consts = activated_constraints_set(model)
+
+        solver.options['bound_push'] = 1e-20
+        solver.options['mu_init'] = 1e-6
+        model.fs.unit.initialize(optarg=solver.options)
+
+        fin_fixed_vars = fixed_variables_set(model)
+        fin_act_consts = activated_constraints_set(model)
+
+        assert degrees_of_freedom(model) == 0
+
+        assert len(fin_act_consts) == len(orig_act_consts)
+        assert len(fin_fixed_vars) == len(orig_fixed_vars)
+
+    @pytest.mark.component
+    def test_solve(self, chlorination_obj):
+        model = chlorination_obj
+        solver.options['max_iter'] = 2
+        results = solver.solve(model)
+        assert results.solver.termination_condition == TerminationCondition.optimal
+        assert results.solver.status == SolverStatus.ok
+
+    @pytest.mark.component
+    def test_solution(self, chlorination_obj):
+        model = chlorination_obj
+
+        assert pytest.approx(300.0016, rel=1e-3) == value(model.fs.unit.outlet.temperature[0])

```

Comment:
I would generally suggest a tolerance of `rel=1e-5` for most tests (solver tolerance is at least 1e-6), unless there is an `eps` involved.